### DISCUSSION TITLE
Trigger per termostato.

### DISCUSSION THREAD

**Discussion by duedipicche** [indomus/forum#2933] [HUB: Home Assistant]
Esiste un trigger, o qualsiasi cosa, che possa usare per creare un'eventuale automazione quando viene usato ul cursore della card "Termostato"?
Vorrei "catturare" ogni eventuale spostamento del cursore che metto in foto per poi rimetterlo nelle posizione da me prefissata.
So come fare l'automazione, ma non trovo da nessuna parte il trigger che cerco.
Grazie per l'eventuale attenzione e, se non sono stato chiaro, chiedetemi pure.
Vittorio
![thermo](https://github.com/indomus/forum/assets/38075965/2d462f4d-e4c5-437e-8159-e65e9229e328)

**Comment by snakuzzo** [comment]
Guarda questo screenshot

![image](https://github.com/indomus/forum/assets/7205371/020046e0-4b47-446e-a346-3286bd28ac50)

La temperatura impostata equivale all'attributo `temperature`, quindi ti basta usare come trigger lo stato di quell'attributo

Se poi vuoi beccare proprio l'azione "manuale" di qualcuno che va sulla dashboard di Home Assistant a cambiare la temperatura target, devi metterti in ascolto sugli eventi (sotto de tools) e creare un trigger basato sugli eventi (https://www.home-assistant.io/docs/automation/trigger/#event-trigger)

**Comment by duedipicche** [comment]
Grazie ci proverò.
